package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "prebuild": "cp generated/openapi-client/openapi.yml public/openapi-client.yml && cp generated/openapi-server/openapi.yml public/openapi-server.yml",
     "build": "next build",
-    "dev": "next dev",
+    "dev": "npm run infra:dev:up && next dev && npm run infra:dev:down",
```

Comment:
I like the current behavior as I mostly have the infra docker compose running in the background forever. This also breaks `npm run dx` which assumes that the db is already running.